bodhi/server/templates/update.html

```diff
@@ -48,6 +48,51 @@
 
     // These are the required taskotron tests
     var requirements = ${update.requirements_json | n};
+    // show the Greenwave decision
+    var greenwave_api_url = '${request.registry.settings["greenwave_api_url"]}';
+    var missing_tests = {};
+    // handle Greenwave decision
+    var handle_unsatisfied_reqs = function(data){
+        $.each(data['unsatisfied_requirements'], function(i, req) {
+            if (req.type == 'test-result-missing') {
+                if (missing_tests[req.item] === undefined)
+                    missing_tests[req.item]= [];
+                missing_tests[req.item].push(req);
+            }
+            // the user may have already specified this in the required taskotron tests
+            if ($.inArray(req.testcase, requirements) == -1) {
+                requirements.push(req.testcase);
+            }
+        });
+    };
```

Comment:
@matt8754 after watching your [demo](https://fedorapeople.org/groups/factory2/sprint-035/mjia-bodhi-greenwave.mp4) I see what you mean. It seems that you've done some nice user feedback there that I do think is helpful and useful. I suppose we could take the approach you've done here, and then refactor it later when we refactor the resultsdb code to keep the data in Bodhi's backend rather than live querying resultsdb.

@ryanlerch @kparal @AdamWill, what do you think?

So yeah, point #1 that I made above will still be confusing until we do that refactor. As for point #2, what I meant is that every time anyone loads a Bodhi update, Greenwave will be queried where it would only be queried once and cached on Bodhi's server if we keep this information on Bodhi's backend. For example, say there's a Firefox update and people load the page about it 100 times. With this code, that will cause 100 queries on the Greenwave server. If we refactor Bodhi to be able to store this information in its backend, that would result in 1 query instead. This has caused considerable load on ResultsDB in the past (though I think some of that problem has been addressed by making this JS more efficient than it used to be).

Anyway, I'm not opposed to doing it in JavaScript for now, but I would like feedback from @ryanlerch, @kparal, or @AdamWill since they have been heavy contributors to this code recently.